### DISCUSSION TITLE
[Examples] Provide example of using Pulumi to deploy NextJS app on AWS

### DISCUSSION THREAD

**Discussion by thesobercoder** [vercel/next.js#62358] [Help]
# Feature request

Please provide an example of how to use Pulumi to deploy NextJS to AWS replicating how deployments work on Vercel infrastructure with edge network.

I'm looking for an example which replicates the same behavior as this [component](https://github.com/serverless-nextjs/serverless-next.js) does for `serverless`.

**Comment by zvictor** [comment]
Pulumi is becoming more popular and can't be ignored.
It came to a point that I'd rather move out of Vercel than move out of Pulumi 🥹

**Comment by sladg** [comment]
Pulumi is not standardised and it will take couple of years to be.
If you are looking for AWS, I offer a solution: https://github.com/sladg/nextjs-lambda

Serverless framework is not really and option anymore I would say. They are struggling with keeping up with changes and support between major versions is rather limited.
